examples/http/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "http-example",
```

Comment:
Is the package won't be published to NPM, should it have a name? Or is there another way to communicate that a package isn't published?

Reply:
Yes, this example package won't be published to NPM.